### DISCUSSION TITLE
The fish redirect manifesto

### DISCUSSION THREAD

**Discussion by abelcha** [fish-shell/fish-shell#10865] [General]
im sure many of already looked up how to combine stderr and stdout, its one of the top stackoverflow question

when you look a the bash equivalent `2>&1 ` , its hard to remember and difficult to understand for and outsider. 

the '&' character can easily lead to having a process silently running in the background, and wrinting somewhere

Enters the fish redirect
```
echo whatever ><> test.txt
```
Visually understandable, you combine the streams >< and redirect > , and only use the classical unix redirect characters

Theres been few discussion in the pass, but was in the end deemed too rare to imply a significant change

But looking a this also as a "marketing" tool, i think it might be worth the change, 
if someone havent heard about the fish shell, just showing this trick is a memorable way of demonstrating fish's philosophy of making a classical 90's shell, without the arcane syntax
wyt

**Comment by faho** [comment]
This looks cute, but in all seriousness I don't think we want it.
